resteasy-client-api/src/main/java/org/jboss/resteasy/client/jaxrs/ResteasyClientBuilder.java

```diff
@@ -50,6 +50,8 @@ public enum HostnameVerificationPolicy {
      */
     public static final String PROPERTY_PROXY_SCHEME = "org.jboss.resteasy.jaxrs.client.proxy.scheme";
 
+    public static final String PROPERTY_FOLLOW_REDIRECTS = "org.jboss.resteasy.jaxrs.client.follow-redirects";
```

Comment:
We have started to change the prefix we use. This should be something like `dev.resteasy.client.follow.redirects`.